t player1RanRolls = []

```diff
@@ -0,0 +1,65 @@
+[33mcommit 89277cdd06ad8f2f768f05ed1a3c32cd8792102e[m[33m ([m[1;36mHEAD -> [m[1;32mmain[m[33m)[m
```

Comment:
Why is this file in your repo? Please remove unnecessary files before committing. The only file that should be changed is `script.js`